gitlab/client.go

```diff
@@ -76,14 +78,14 @@ func FetchProjects() ([]gitlab.Project, error) {
 
 func FetchCommitsForProject(projectID int) ([]gitlab.Commit, error) {
 	var allCommits []gitlab.Commit
-	baseURL := fmt.Sprintf("%s/projects/%d/repository/commits", gitlabApiUrl, projectID)
+	baseURL := fmt.Sprintf("%s/projects/%d/repository/commits?author=%s", gitlabApiUrl, projectID, gitlabUsername)
```

Comment:
Thanks for the MR.

One comment/question, the `gitlabUsername` is optional, if empty will the request work fine? Otherwise, you could add on an `if` condition if present.

Reply:
I have verified it myself.